src/templates/core/category_list.html

```diff
@@ -1,44 +1,55 @@
-{% extends "base/base.html" %}
+{% extends "base/base_card.html" %}
```

Comment:
I'm not sure that we should use another base block for these forms, as I remember,  "main" block already has a base structure for nested form, anyway, we should discuss it.

Reply:
If we would like use white card behind, we need use new extends. There is the difference.
![Screenshot from 2021-08-16 12-47-17](https://user-images.githubusercontent.com/45559907/129545825-e382d6cd-f180-4a22-bd11-ba1010dfbee0.png)